src/parser/abstractparser.cpp

```diff
@@ -465,6 +465,7 @@ QString AbstractParser::normalizeStringCopy(QString string)
     // transliterations to accidentally count as ANSI.
     ParserUtils::removeAnsiMarksInPlace(string);
     ParserUtils::toAsciiInPlace(string);
+    string.remove(C_CARRIAGE_RETURN);
```

Comment:
Wont this change any regexp's or lines sent to the user?

Reply:
Possible but I think it's only lines sent for internal parsing. I'll audit it further.

Reply:
I confirmed this doesn't impact what is sent to the user. It looks like for internal actions the logic is unchanged too. Risk is low :-)